tests/test_fs_jail.c

```
@@ -18,10 +18,42 @@
+#include <dirent.h>
 
 /* g_fs_jail_root is a non-static global defined in fs_jail.c */
 extern char g_fs_jail_root[];
 
+static void rmrf_dirpath(const char *path) {
+    DIR *d = opendir(path);
+    if (!d) {
+        unlink(path);
+        return;
+    }
+    struct dirent *e;
```

Clang (14.0.6)

[warning] variable 'e' is not initialized

(cppcoreguidelines-init-variables)